config/routes.rb

```diff
@@ -364,6 +364,7 @@
       get '/:step' => 'inherited_proofing#show', as: :step
       put '/:step' => 'inherited_proofing#update'
       get '/return_to_sp' => 'inherited_proofing#return_to_sp'
+      get '/errors/no_information' => 'inherited_proofing#no_information'
```

Comment:
Even if there's no logic associated with the route, conventionally it's preferred to at least include an empty method for the action in the controller, so that it's more obvious what actions exist when looking at the controller implementation.

```rb
class InheritedProofingController
  # ...

  def no_information; end
end
```

Reply:
I tried testing this locally by going directly to http://localhost:3000/verify/inherited_proofing/errors/no_information, only to see a 500 error `undefined method `uuid' for nil:NilClass`. It seems that we need to be adding some validation in the base controller to avoid 500.

Reply:
@aduth I actually first added that method to controller and then removed it thinking that preferred. Ex. no_camera action on doc_auth/errors . I will re add it.

Reply:
We have a future tasks for adding before_filters but I added before_action :confirm_two_factor_authenticated to redirect away from that